ic-agent/src/lib.rs

```diff
@@ -103,11 +103,12 @@
 //! generally available, additional details about the versions supported will
 //! be available here.
 
-#![deny(
+#![warn(
```

Comment:
Intentionally left in?

Reply:
Yes. I had been meaning to remove it, since it makes development more annoying and CI still covers it with `-Dwarnings`.